sources/editor/Stride.GameStudio.Avalonia/Helpers/StrideGameStudio.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Stride.GameStudio.Avalonia.Helpers;
+
+public class StrideGameStudio
+{
+    public static string EditorVersion => StrideVersion.NuGetVersion;
+    public static string EditorVersionMajor => new Version(StrideVersion.PublicVersion).ToString(2);
+    public static string AnswersUrl => "https://gamedev.stackexchange.com/tags/stride"; // #706
```

Comment:
Is it active? Do people from our community moderate it?

If not, I'd rather we link to Discord.

Reply:
I'm not sure, but at first look there is not much traffic, I've added discord link 👍